TrackletAlgorithm/MemoryTemplateBinnedCM.h

```diff
@@ -46,6 +46,8 @@ class MemoryTemplateBinnedCM{
 
   ap_uint<8> binmask8_[kNBxBins][8];
   ap_uint<32> nentries8_[kNBxBins][8];
+  ap_uint<32> nentries8A_[kNBxBins*8];
+  ap_uint<32> nentries8B_[kNBxBins*8];
```

Comment:
@jasonfan393 Is there a reason these are not two-dimensional arrays, like the other data members are?

Reply:
Sounds good to me.